include/zephyr/misc/lorem_ipsum.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2015 Intel Corporation.
+ * Copyright (c) 2024 Tenstorrent AI ULC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_MISC_LOREM_IPSUM_H_
+#define ZEPHYR_MISC_LOREM_IPSUM_H_
+
+#include <zephyr/toolchain.h>
+
+/*
+ * N.B. These strings are generally only used for tests and samples. They are not part of any
+ * official Zephyr API, but were moved to reduce duplication.
+ */
+
+/* Generated by http://www.lipsum.com/
+ * 1 paragraph, 69 words, 445 bytes of Lorem Ipsum
+ */
+#define LOREM_IPSUM_SHORT                                                                          \
+	"Lorem ipsum dolor sit amet, consectetur adipiscing elit, "                                \
+	"sed do eiusmod tempor incididunt ut labore et dolore magna "                              \
+	"aliqua. Ut enim ad minim veniam, quis nostrud exercitation "                              \
+	"ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis "                           \
+	"aute irure dolor in reprehenderit in voluptate velit esse "                               \
+	"cillum dolore eu fugiat nulla pariatur. Excepteur sint "                                  \
+	"occaecat cupidatat non proident, sunt in culpa qui officia "                              \
+	"deserunt mollit anim id est laborum."
+
+#define LOREM_IPSUM_SHORT_STRLEN 445
+BUILD_ASSERT(sizeof(LOREM_IPSUM_SHORT) == LOREM_IPSUM_SHORT_STRLEN + 1);
+
+/* Generated by http://www.lipsum.com/
+ * 2 paragraphs, 178 words, 1160 bytes of Lorem Ipsum
+ */
+#define LOREM_IPSUM                                                                                \
+	"Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque "                        \
+	"sodales lorem lorem, sed congue enim vehicula a. Sed finibus diam sed "                   \
+	"odio ultrices pharetra. Nullam dictum arcu ultricies turpis congue, "                     \
+	"vel venenatis turpis venenatis. Nam tempus arcu eros, ac congue libero "                  \
+	"tristique congue. Proin velit lectus, euismod sit amet quam in, "                         \
+	"maximus condimentum urna. Cras vel erat luctus, mattis orci ut, varius "                  \
+	"urna. Nam eu lobortis velit."                                                             \
+	""                                                                                         \
```

Comment:
This should be `\n` if we want to have two paragraphs. Now all the strings are just cat together, and there is even no space between ` velit.Nullam ` strings. Dunno if this makes any practical difference to anything, just if one prints the output and tries to read it, so I am totally fine with this too.

Reply:
Fixed - I just regenerated a new one version specifying 1159 words (so the additional `\n` made it 1160).

I feel like that number rings a bell (maybe the payload of an ipv6 packet minus a couple of headers?)

Anyway, the newline is there now.